src/network-analytics/azext_network_analytics/tests/latest/test_network_analytics.py

```diff
@@ -70,6 +70,6 @@ def test_data_product_e2e(self, resource_group, location, user_id="test@microsof
 
         #List DP
         dplistinrg = self.cmd('az network-analytics data-product list --resource-group {group}').get_output_in_json()
-        self.assertEquals(len(dplistinrg), 0, msg="Data product list is not empty after deletion")
+        self.assertEqual(len(dplistinrg), 0, msg="Data product list is not empty after deletion")
 
-    pass
\ No newline at end of file
+    pass
```

Comment:
Missing ending line

Reply:
I don't understand. This old file is missing ending line, not the new file.